tests/testthat/_snaps/buffer.md

```diff
@@ -0,0 +1,55 @@
+# bad args
+
+    Code
+      spatial_clustering_cv(ames_sf, buffer = 50)
+    Condition
+      Error in `buffer_indices()`:
```

Comment:
this would be nice to propagate up so that it says it's an error in `spatial_clustering_cv()`